src/components/PeoplePage.tsx

```diff
@@ -1,8 +1,60 @@
+import React, { useEffect, useState } from 'react';
 import { PeopleFilters } from './PeopleFilters';
 import { Loader } from './Loader';
 import { PeopleTable } from './PeopleTable';
+import { getPeople } from '../api';
+import { Person } from '../types';
+import { useSearchParams } from 'react-router-dom';
 
 export const PeoplePage = () => {
+  const [people, setPeople] = useState<Person[]>([]);
+  const [loading, setLoading] = useState(true);
+  const [error, setError] = useState<string | null>(null);
+
+  useEffect(() => {
+    const fetchPeople = async () => {
+      try {
+        setLoading(true);
+        const data = await getPeople();
+
+        setPeople(data);
+        setLoading(false);
+      } catch (e) {
+        setError('Something went wrong');
```

Comment:
Consider providing more detailed error information to the user, if available. This can help users understand what went wrong and potentially how to fix it. You might want to log the actual error message or display it conditionally.